src/main/app.ts

```diff
@@ -77,6 +78,9 @@ if (toBoolean(config.get<boolean>('featureToggles.testingSupport'))) {
   new TestingSupportFeature().enableFor(app)
 }
 
+if (app.settings.nunjucksEnv && app.settings.nunjucksEnv.globals) {
```

Comment:
why add this here?
we do it everywhere in nunjucks/index.ts

Reply:
No index file for the shared elements. We could create `core` feature or smash `index.ts` in the root directory. We should also move application paths form `app` directory as it does not follow structure convention.

Shall we do it in follow up PR to separate the refactoring? Makes little sense to add more to the PR since it was already reviewed.

Reply:
not sure what you mean can't you add the path here:
https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/modules/nunjucks/index.ts#L103

Reply:
Oh I could but I do not think this is the best place. I know we started putting all globals there IMO features / modules should register objects they introduce with nunchucks engine.

Reply:
sure, but they are all there now and this change doesn't look to be the time to change it.
please move it